blocks/feed/feed.js

```diff
@@ -76,14 +76,24 @@ const resultParsers = {
         }
       });
       if (cardImage) {
-        row.push(cardImage);
+        if (result.path) {
+          const pathImg = document.createElement('a');
+          pathImg.href = result.path;
+          pathImg.append(cardImage);
+          row.push(pathImg);
+        } else row.push(cardImage);
+      }
+      if (result.featured === 'true' && block.classList.contains('featured')) {
```

Comment:
`featured` must be read from `blockCfg`

Reply:
This featured is not of the one in blockCfg. This featured is of the block name - "feed (featured)", since there are certain unique changes to be done for latest featured card & not to every featured card.

Reply:
With regards to https://github.com/hlxsites/sunstar-foundation/pull/55#discussion_r1431000125 - this is all copied as it is from the hightlight.js from sunstar - https://github.com/hlxsites/sunstar/blob/main/blocks/highlight/highlight.js - just to make consistency across the blocks.